frontend/src/components/material/MaterialTable.vue

```diff
@@ -252,6 +258,9 @@ export default {
 
     // period Entity for displaying material items within a period (should be null if materialNode is provided)
     period: { type: Object, required: false, default: null },
+
+    // Disable the filter button activity / period
+    disablePeriodFilter: { type: Boolean, required: false, default: false },
```

Comment:
This component now has an attribute `disablePeriodFilter` and a computed `periodFilterEnabled`. It took me over 5 minutes to understand why.

Counter suggestion: `hidePeriodFilter` says better what it does.

Reply:
You have right the naming is not really happy
I changed it to hidePeriodFilter